Update blitz config docs and add info about log type (#597)
blitz-js/blitzjs.com: app/pages/docs/blitz-config.mdx

}
```

### Log Type {#log-type}

> It is also possible to adjust the log type provided by Blitz, which 
> could be useful if you prefer to forward the log in a more usable
> format like JSON.

Reviewer: ```suggestion
It is also possible to adjust the logging type, which could 
be helpful if you prefer to forward the logs in JSON format.
```
